packages/web/src/scss/components/Box/index.html

```diff
@@ -52,6 +52,19 @@ <h2 class="docs-Heading">With Background Color</h2>
   </div>
 </section>
 
+<section class="py-900 py-tablet-1000">
+  <div class="Container">
+    <h2 class="docs-Heading">With Background Gradient</h2>
+    <div class="docs-Stack docs-Stack--start">
+
+      <p>For demo purposes, the boxes have custom padding.</p>
+      <div class="bg-gradient-primary p-800">Primary Background Gradient</div>
```

Comment:
Non-related to this PR: `p-<space-token>` works, but `m-<space-token>` not? Why?

Reply:
There is no use case known.